recipes/llnl-units/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+from conans import ConanFile, tools
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.files import copy, rm, rmdir
+
+
+class UnitsConan(ConanFile):
+    name = "llnl-units"
+    license = "BSD-3"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://units.readthedocs.io"
+    description = (
+        "A run-time C++ library for working with units "
+        "of measurement and conversions between them "
+        "and with string representations of units "
+        "and measurements"
+    )
+    topics = (
+        "units",
+        "dimensions",
+        "quantities",
+        "physical-units",
+        "dimensional-analysis",
+        "run-time",
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "base_type": ["uint32_t", "uint64_t"],
+        "namespace": "ANY",
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
```

Comment:
Please check [this methods implemented in the cmake-project template](https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L61-L67), they are important to properly handle fpic behaviour in the different configurations